sql/char_fishing.sql

```diff
@@ -0,0 +1,22 @@
+-- --------------------------------------------------------
+-- Add history for player fishing data
+-- --------------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS `char_fishing` (
```

Comment:
All the other definitions are DROP then CREATE

Reply:
Would that not force the table to be dropped anytime someone ran the file even accidentally?  I thought that was only used for static data?  Does it just get added to the "protected" list in dbtool then?

Reply:
Add them to the protected table in dbtool and it'll handle making sure there's no overwrites. I'm actually curious _why_ why we were dropping the tables in the first place and not just using `IF NOT EXISTS` like here. It doesn't even seem to be consistent across all protected tables. Either way, adding it to dbtool helps with the "lite backup" option.